scripts/migrate-build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Could you also add to README an explanation of how these scripts are run on TC, now that it's less obvious!